frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -342,6 +357,25 @@ export const useMutateFormSettings = () => {
     },
   )
 
+  // TODO: fix the TS typing
+  const mutateFormWhitelistSetting = useMutation<any, Error, any, any>(
```

Comment:
Can the generics be removed? TS should automagically infer them.

Reply:
removed